src/main/java/no/digipost/security/cert/CertificateValidator.java

```diff
@@ -162,6 +162,11 @@ private CertStatus ocspLookup(TrustedCertificateAndIssuer certificateAndIssuer,
                             return UNDECIDED;
                         }
 
+                        if(basix == null) {
+                            LOG.warn("OCSP from {} for certificate {}, returned a null response", ocspResult.uri, describe(certificate));
```

Comment:
Jag blir osäker på om inte null egentligen betyder GOOD eftersom en bouncycastle-utvecklare skriver

"CertificateStatus.GOOD actually equals null and getCertStatus() if the certificate status is good actually returns null.

The idea was that people would be able to write:

if (resp.getCertStatus() != CertificateStatus.GOOD)
{
      // handle issue...
}

without having to use .equals().

Apologies for the confusion, but it seemed like the simplest approach."

Men samtidigt så har vi gjort exakt samma fix i offentlig under Stefans och Gustavs inrådan

https://github.com/digipost/digipost-offentlig/commit/b7c69d7355034187be9c5023cc45ea8bf5a43c9e